spec/views/cases/offender_sar/_case_details_html_slim_spec.rb

```diff
@@ -60,6 +60,23 @@ def login_as(user)
       expect(partial.third_party_name.data.text).to eq "Rick Westor"
       expect(partial.requester_reference.data.text).to eq "FOOG1234"
       expect(partial.third_party_company_name.data.text).to eq "Foogle and Sons Solicitors at Law"
+      expect(partial.third_party_email.data.text).to eq "foogle@solicitors.com"
+    end
+
+    it "displays rejected details if present" do
+      rejected_case = (create :offender_sar_case, :rejected, rejected_reasons: "ID required, Court data request, further identification, police data").decorate
+      assign(:case, rejected_case)
+      render partial: "cases/offender_sar/case_details", locals: {
+        case_details: rejected_case,
+        link_type: nil,
+        allow_editing: true,
+      }
+
+      partial = offender_sar_case_details_section(rendered)
+
+      # expect(partial.has_selector?("rejected")).to eq true
+      expect(partial.has_selector?(".rejected_reasons")).to eq(true)
+      expect(partial.rejected_reasons.data.text).to eq "ID required, Court data request, further identification, police data"
```

Comment:
You've called it `rejected_reason` in the site_prism file.

Reply:
site_prism file?